containers/unit-tests/run.sh

```diff
@@ -1,5 +1,10 @@
 #!/bin/bash
 
+if [ -z "${TEST_SCENARIO:-}"]; then
+    echo 'Required variable TEST_SCENARIO not set'
```

Comment:
The script uses `set -eu` a bit further down, that would cause a very similar error message already. Let's just move up the `set -u`, or possibly even the `set -e`?

Reply:
I hit `TEST_SCENARIO` being undefined on a previous push because `start` was setting the `SCENARIO` variable instead.  It took a while before the fail, so I added this because I wanted the failure to come early.

Reply:
Sure, but I meant that `set -u` does exactly that for you. And I think we can easily move it up. Anyway, I'm happy to do that in a follow-up.